WordPress/src/androidTest/java/org/wordpress/android/e2e/pages/MySitesPage.java

```diff
@@ -87,6 +88,11 @@ public StatsPage clickStats() {
                 onView(withId(R.id.tabLayout))
         );
 
+        // Stats are opened with the last selected tab active (or `Insights` by default)
+        // Since we don't know what was the last used tab, we're using a "dumb" wait
+        // instead of the conditional wait (e.g. waiting for a certain tab element to appear).
```

Comment:
The reasoning for the hardcoded sleep.

Reply:
Oh no, my bad! Sure, I intended not for 2 ms, but 2 seconds. Just changed it to `WPSupportUtils.sleep()`. Thank you for spotting this in time - this could have potentially gone very wrong, if the stats tests continue to fail.

Reply:
You are super fast, thank you for [updating it](https://github.com/wordpress-mobile/WordPress-Android/pull/16799/commits/b04f544d55b4aa765c40cd3f7445f99ef5be36fd) Sergiy! 🥇